java/test/jmri/jmrix/roco/z21/Z21XNetTurnoutManagerTest.java

```diff
@@ -13,6 +15,16 @@
  */
 public class Z21XNetTurnoutManagerTest extends jmri.jmrix.lenz.XNetTurnoutManagerTest {
 
+    @Test
+    @Override
+    public void testSetAndGetOutputInterval() { // Z21/XNetTurnoutManager has no direct access to Memo, ask TC
```

Comment:
Every Manager has direct access to its memo via `Manager.getMemo()`, so I don’t understand the comments in this code.

Reply:
Will review this. Back when I wrote this, that was not always the case. Even now, some tests still do not configure like an adapter does.